src/query/api/v1/handler/prometheus/native/list_tags.go

```diff
@@ -48,13 +48,15 @@ var (
 
 // ListTagsHandler represents a handler for list tags endpoint.
 type ListTagsHandler struct {
-	storage storage.Storage
-	nowFn   clock.NowFn
+	storage             storage.Storage
```

Comment:
This new limit wont affect aggregate endpoints will it?

Reply:
It will too yeah.

Reply:
But will correctly respect the size of the aggregate values, not the amount of documents scanned.